src/arch/i386/Makefile

```diff
@@ -95,6 +95,8 @@ SYSLINUX_DIR_LIST	:= \
 	/usr/share/syslinux/modules/bios \
 	/usr/local/share/syslinux \
 	/usr/local/share/syslinux/bios \
+	/usr/local/share/syslinux/bios/core \
+	/usr/local/share/syslinux/bios/com32/elflink/ldlinux \
```

Comment:
Merged (in the replacement `util/genfsimg` as https://github.com/ipxe/ipxe/commit/9557497e7

Reply:
That was on my fork locally for a long time.